bin/common/install_tools.sh

```diff
@@ -53,8 +53,17 @@ if systemctl list-unit-files kube-apiserver.service | grep --quiet enabled ; the
     sleep 1
   done
 
-  echo "Installing tiller for helm ..."
-  ${do_as_vagrant} helm init
+  echo "Installing tiller for helm with service account ..."
+  tiller_sa_namespace="kube-system"
+  tiller_sa_name="tiller"
+  if ! kubectl get serviceaccount "${tiller_sa_name}" --namespace "${tiller_sa_namespace}" 1> /dev/null 2> /dev/null; then
+    kubectl create serviceaccount "${tiller_sa_name}" --namespace "${tiller_sa_namespace}"
+  fi
+  tiller_crb_name="tiller"
+  if ! kubectl get clusterrolebinding "${tiller_crb_name}" --namespace "${tiller_sa_namespace}" 1> /dev/null 2> /dev/null; then
```

Comment:
I don't think cluster role bindings are within a namespace

Reply:
Right. Why would it!?